docs/ADR-67-importing-wearables.md

```diff
@@ -0,0 +1,206 @@
+- The `model` is the path to the model of the wearable. This implies that the wearable can have only one model for all representations.
+- The `bodyShape` is the body shape of the wearable. This can only have one value, either `male`, `female` or `both`, implying
+
+The current format has some downsides that we need to tackle:
+
+1. It doesn't have the possibility to define representations with different 3D models, that is, a female and a male representation, each one with its 3D model shaped to their body shape.
+2. Wearable tags can't be defined.
+3. It's not possible to define which categories get to be hidden or replaced by the wearable and, as it doesn't allow each representation to be detailed, it doesn't allow overriding hides and replaces for each of representation.
+4. The collection where the wearable is created into can't be specified.
+5. The name of the file `asset.json` is confusing as assets already have a file with the same name on them.
+
+These downsides make the current `asset.json` configuration file not suitable for importing wearables.
+
+## Proposed solution
+
+To solve the downsides described above, two changes are proposed:
+
+1. The configuration file (former `asset.json`) should be named `wearable.json`.
+2. A new format, with an upgraded schema.
+
+The format proposed is the following:
+
+```typescript
+{
+  type: "object",
+  properties: {
+    id: { type: "string", format: "uuid" },
+    collectionId: { type: "string", format: "uuid" },
+    name: { type: "string" },
+    description: { type: "string" },
+    urn: { type: "string" },
+    rarity: { $ref: "#/$defs/rarity" },
+    category: { $ref: "#/$defs/category" },
+    hides: {
+      type: "array",
+      items: { $ref: "#/$defs/category" },
+    },
+    replaces: {
+      type: "array",
+      items: { $ref: "#/$defs/category" },
+    },
+    tags: { type: "array", items: { type: "string" } },
+    representations: {
+      type: "array",
+      items: { $ref: "#/$defs/wearable-representation" },
+    },
+  },
+  required: ["name", "rarity", "category", "representations"],
+  additionalProperties: false,
+  $defs: {
+    rarity: {
+      type: "string",
+      enum: [
+        "unique",
+        "mythic",
+        "legendary",
+        "epic",
+        "rare",
+        "uncommon",
+        "common",
+      ],
+    },
+    category: {
+      type: "string",
+      enum: [
+        "eyebrows",
+        "eyes",
+        "facial_hair",
+        "hair",
+        "head",
+        "body_shape",
+        "mouth",
+        "upper_body",
+        "lower_body",
+        "feet",
+        "earring",
+        "eyewear",
+        "hat",
+        "helmet",
+        "mask",
+        "tiara",
+        "top_head",
+        "skin",
+      ],
+    },
+    "wearable-representation": {
+      type: "object",
+      properties: {
+        bodyShapes: {
+          type: "array",
+          items: {
+            type: "string",
+            enum: ["both", "female", "male"],
```

Comment:
uh this is already an array, so having `both` in the enum doesn't make much sense, instead it would be an array with `male` and `female`. Also, instead of using plain `female` and `male`, maybe we should use the actual urn of the body shape ([link](https://github.com/decentraland/common-schemas/blob/main/src/platform/wearables/wearable-body-shape.ts#L8-L11)), which is what is used by the `WearableRepresentation` type ([link](https://github.com/decentraland/common-schemas/blob/main/src/platform/wearables/representation.ts#L11)), that looks like: `urn:decentraland:off-chain:base-avatars:BaseMale`

We already have 3 **different** schemas for representations ([1](https://github.com/decentraland/common-schemas/blob/main/src/platform/wearables/representation.ts#L13), [2](https://github.com/decentraland/common-schemas/blob/main/src/dapps/preview/representation-definition.ts#L7), [3](https://github.com/decentraland/kernel/blob/62dcd0f5f971198490a2630332342bd78b70dbb4/packages/shared/catalogs/types.ts#L75)) lets not create a 4th one 😅

Reply:
You're right! the type is wrong, it should be a string with any of those values ('both', 'female' or 'male'). The rationale is to have something less verbose for the users to write when writing this file to import their wearables.
I'll change it to be a string.